pilot/pkg/kube/inject/inject.go

```diff
@@ -64,6 +65,7 @@ var (
 	annotationRegistry = map[string]annotationValidationFunc{
 		annotations.Register(annotationPolicy, "").Name:                                        alwaysValidFunc,
 		annotations.Register(annotationStatus, "").Name:                                        alwaysValidFunc,
+		annotations.Register(annotationRewriteAppProbers, "").Name:                             validateBool,
```

Comment:
annotations.Register() second argument is the description of the annotations. I assume @geeknoid intention is to auto generate the document at some point. Previous annotation does not have doc, but let's add for new one.

Some brief sentence about what this annotation is for is good enough.

Reply:
added description. please let me know if you have better desc